lib/booking/ui/calendar.dart

```diff
@@ -91,124 +92,117 @@ class Calendar extends HookConsumerWidget {
       }
     }
 
```

Comment:
I removed the LayoutBuilder widget because it did not seem useful to me. The SfCalendar chooses by default to take the maximum of avaible space.